README-AIRS.md

```diff
@@ -0,0 +1,122 @@
+# AKS Secure Baseline Hack (internal)
+
+## Welcome to the Patterns and Practices (PnP) AKS Secure Baseline (ASB) hack!
+
+- This repo contains modifications and additions to tailor it specifically for hack events and **should not be used for production deployments**
+  - Original PnP AKS Secure Baseline repo is located [here](https://github.com/mspnp/aks-secure-baseline)
```

Comment:
I modified order, format, and slight word changes for the first bullet and subbullet. Please check.